src/main.zig

```diff
@@ -48,6 +48,37 @@ pub const EmbeddedExecutable = struct {
             }
         } else @panic("app package not found");
     }
+
+    pub fn addPackagePath(exe: *EmbeddedExecutable, name: []const u8, pkg_index_path: []const u8) void {
```

Comment:
sorry for the late review! btw you could simplify and just call `exe.inner.addPackagePath()`

Reply:
Hi, no problem. Unfortunately, the added package can't be found if you call `inner.exe.addPackagePath()` directly.

Example:
```
./src/main.zig:3:14: error: unable to find 'zbor'
const cbor = @import("zbor");
```
I use Linux and Zig version `0.10.0-dev.3685+dae7aeb33`.

Reply:
ooh yes, I forgot about the package structure, you did the correct thing